frontend/packages/local-storage-operator-plugin/src/models.ts

```diff
@@ -0,0 +1,14 @@
+import { K8sKind } from '@console/internal/module/k8s';
+
+export const AutoDetectVolumeModel: K8sKind = {
+  label: 'Volume',
+  labelPlural: 'Volumes',
+  apiVersion: 'v1alpha1',
+  apiGroup: 'local.storage.openshift.io',
+  plural: 'volumes',
+  abbr: 'ADV',
+  namespaced: true,
+  kind: 'AutoDetectVolume',
+  id: 'autodetectvolume',
+  crd: true,
+};
```

Comment:
new line

Reply:
Whats the difference in `LocalVolumeDiscovery` CR and this one ?
Arent we supposed to add that ?

Reply:
`LocalVolumeDiscovery` is required to auto detect the local devices on nodes, which is not part of this release 4.5.